### PR TITLE
napalm_ping.py

### PR DESCRIPTION
**Pull request by tejasmokashi1992**
Add napalm_ping plugin to nornir

### CODE REVIEW

**Comment by tejasmokashi1992** [comment]
Script napalm ping
Output

**Comment by tejasmokashi1992** [comment]
image
Image shows two checks failed. I dont understand them. Let me know what can or needs to be done?

**Comment by dbarrosop** [comment]
Hello, thanks for the PR. It's a formatting issue. If you click in them you should see the details close to the bottom. Alternatively, running pylama and/or black locally should give you the same information.

**Comment by tejasmokashi1992** [comment]
@dbarrosop all checks cleared

**Comment by tejasmokashi1992** [comment]
@dbarrosop is there any more info you need from my side ?

**Comment by tejasmokashi1992** [comment]
@dbarrosop i need to use napalm_ping in my other scripts.
Please can you check further & let me know if script is ready to be included in nornir.

**Comment by dmfigol** [comment]
@tejasmokashi1992 
Please keep in mind that all maintainers are working on nornir in their free time and an expectation that we will check issues and PRs daily is completely unrealistic. Even if the PR is accepted, it will still take time for this feature to appear in the new release on PyPi (and currently we haven't even discussed 2.4 release).
There is nothing stopping you from using this task in your code right now, as tasks can live in user code, they don't touch nornir internals.

File: nornir/plugins/tasks/networking/napalm_ping.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    timeout: Optional[int] = 2,
+    size: Optional[int] = 100,
+    count: Optional[int] = 5,
+    vrf: Optional[str] = None,
+) -> Result:
+    """
+    Executes ping on the device and returns a dictionary with the result.
+
+    Arguments:
+      dest(destination) – Host or IP Address of the destination. (type=string)
```
Comment (dbarrosop) [inline nornir/plugins/tasks/networking/napalm_ping.py]: Can you format this consistently with the other plugins? For instance:

https://github.com/nornir-automation/nornir/blob/develop/nornir/plugins/tasks/networking/netmiko_save_config.py

**Comment by dbarrosop** [comment]
The last thing missing is that you have to add the plugin to the file `nornir/nornir/plugins/tasks/networking/__init__.py`

File: nornir/plugins/tasks/networking/napalm_ping.py (hunk 1 above)
Comment (tejasmokashi1992) [reply nornir/plugins/tasks/networking/napalm_ping.py]: @dbarrosop  i have made changes according to format.

**Comment by tejasmokashi1992** [comment]
@dbarrosop i have created a PR #477 for including this plugin in  
nornir/nornir/plugins/tasks/networking/__init__.py

**Comment by dbarrosop** [comment]
Can you close #477 and include the change in this one?

**Comment by tejasmokashi1992** [comment]
@dbarrosop closed #477 PR & added required changes in
nornir/nornir/plugins/tasks/networking/init.py

**Comment by dbarrosop** [comment]
Thanks

**Pull request merged**